.rubocop.yml

```diff
@@ -1,23 +1,21 @@
 inherit_from:
   - .rubocop_todo.yml
 
-require:
-  - rubocop-performance
-  - rubocop-rails
-  - rubocop-minitest
+inherit_gem:
+  theforeman-rubocop:
+    - strictest.yml
 
 AllCops:
   TargetRubyVersion: 2.7
   TargetRailsVersion: 5.2
   Exclude:
-    - 'extras/**/*'
-    - 'locale/**/*'
-    - 'node_modules/**/*'
-    - 'vendor/**/*'
```

Comment:
don't delete that line please, otherwise rubocop tries to scan all gems we install as dependencies which takes a lot of time and produces errors